models/vrp.rb

```diff
@@ -227,6 +228,14 @@ def self.check_consistency(hash)
         end
       end
 
+      if configuration[:restitution]
+        if configuration[:restitution][:geometry].to_a.any? &&
```

Comment:
do we need `to_a` here? isn't geometry an array by default -- can it be nil?

Reply:
It is sometimes in the tests. I can deal with this case in test_helper maybe.

Reply:
yeah, test_helper coerce might be better suited for this if we have tests which have restitution but no geometry in them.

Reply:
https://github.com/Mapotempo/optimizer-api/pull/165/commits/8dd8ebc18be97feb092c2b84697bc632837ceda9